### PR TITLE
Fix: add label to checkbox component

### PR DESCRIPTION
**Pull request by clairehhelms**
## What I did

Adding a label to the Checkbox component for greater accessibility. I wrapped the Checkbox component with our InputLabel component. If a label is specified as a prop when instantiating a Checkbox, it will be visually displayed to the right of the component.

Screen Shot 2022-07-19 at 11 46 18 AM

### CODE REVIEW

**Review by wp-aberg — COMMENTED** [review]
Looks good, just the one tweak to keep the checkbox the correct size

File: ui/checkbox/src/Checkbox.tsx [hunk 1]
```diff
@@ -6,6 +6,8 @@ import { Check, Indeterminate } from "@washingtonpost/wpds-assets";
 
 import * as PrimitiveCheckbox from "@radix-ui/react-checkbox";
 
+import { InputLabel } from "@washingtonpost/wpds-input-label";
+
 const StyledCheckbox = styled(PrimitiveCheckbox.Root, {
```
[the remark this answers is not in the document]
Comment (clairehhelms) [reply ui/checkbox/src/Checkbox.tsx:11]: Thank you!!!

File: ui/checkbox/src/Checkbox.tsx [hunk 2]
```diff
@@ -282,6 +293,7 @@ export const Checkbox = React.forwardRef<HTMLButtonElement, CheckboxInterface>(
           </StyledCheck>
         </StyledIndicator>
       </StyledCheckbox>
+      {props.label}</StyledInputLabel>
```
[the remark this answers is not in the document]
Comment (clairehhelms) [reply ui/checkbox/src/Checkbox.tsx:296]: oops yep

**Review by artmsilva — APPROVED** [review]

**Review by wp-aberg — APPROVED** [review]
🚀

**Comment by clairehhelms** [comment]
Victor approved visuals, merging!

**Comment by clairehhelms** [comment]
Victor approved visuals! Merging and will update docs

**Pull request merged**